### PR TITLE
Feature/process dayoff

### PR DESCRIPTION
**Pull request by serjKim**
Save dayoff

### CODE REVIEW

File: mobile/src/calendar/dayoff-action-button.tsx [hunk 1]
```diff
@@ -23,7 +37,18 @@ export class DayoffActionButton extends Component<DayoffActionButtonProps> {
     }
 
     public onDayoffAction = () => {
-        // TODO: do something
+        if (!this.props.interval && this.props.hoursCredit.hours !== 0) {
```
Comment (santee) [inline mobile/src/calendar/dayoff-action-button.tsx]: what is hoursCredit? The value that we show on top? Why do you check for it being equal to 0 in that case?

File: server/Arcadia.Assistant.Calendar/WorkHours/EmployeeWorkHoursActor.cs [hunk 2]
```diff
@@ -16,7 +16,7 @@ public class EmployeeWorkHoursActor : CalendarEventsStorageBase
         /// Positive values means that these days must be worked out.
         /// Negative means that these can be taked as days off
         /// </summary>
-        private int hoursCredit = 0;
+        private int hoursCredit = -17;
```
Comment (santee) [inline server/Arcadia.Assistant.Calendar/WorkHours/EmployeeWorkHoursActor.cs]: revert

File: mobile/src/calendar/dayoff-action-button.tsx (hunk 1 above)
Comment (serjKim) [reply mobile/src/calendar/dayoff-action-button.tsx]: > what is hoursCredit? The value that we show on top?

it's a model, containg hoursCredit value, and its textual representation - _days + 1/2_. Accordingly, We show textual representation on the top.

Comment (santee) [reply mobile/src/calendar/dayoff-action-button.tsx]: why do you check on hours !== 0?

Comment (serjKim) [reply mobile/src/calendar/dayoff-action-button.tsx]: I thought, 0 means there is neither dayoff nor workout hours, so disable to process.

**Pull request merged**